coc.md

```diff
@@ -26,7 +26,7 @@ Be careful in the words that you choose. We are a community of professionals, an
 - Unwelcome sexual attention.
 - Advocating for, or encouraging, any of the above behavior.
 - Repeated harassment of others. In general, if someone asks you to stop, then stop.
-- If you believe someone is violating the code of conduct, we ask that you report it by emailing [Polly Schandorf](mailto:polly@rubybythebay.org) or [Kara Louie](mailto:kara@rubybythebay.org). Any email received will be kept confidential. You may also reach out to any of the organizers in person.
+- If you believe someone is violating the code of conduct, we ask that you report it by emailing [Polly Schandorf](mailto:polly@rubybythebay.org) or [WHO](mailto:WHO@rubybythebay.org). Any email received will be kept confidential. You may also reach out to any of the organizers in person.
```

Comment:
@seanmarcia @jtu0 who should we put as second CoC contact here?

Reply:
Jennifer (jtu@rubybythebayorg)